src/store/Selectors/GraphData.ts

```diff
@@ -20,47 +21,47 @@ import {
 export const decorateGraphData = (graphData: GraphElements): DecoratedGraphElements => {
   const elementsDefaults = {
     edges: {
-      grpc: 'NaN',
-      grpcErr: 'NaN',
-      grpcPercentErr: 'NaN',
-      grpcPercentReq: 'NaN',
-      http: 'NaN',
-      http3xx: 'NaN',
-      http4xx: 'NaN',
-      http5xx: 'NaN',
-      httpPercentErr: 'NaN',
-      httpPercentReq: 'NaN',
+      grpc: NaN,
+      grpcErr: NaN,
+      grpcPercentErr: NaN,
+      grpcPercentReq: NaN,
+      http: NaN,
+      http3xx: NaN,
+      http4xx: NaN,
+      http5xx: NaN,
+      httpPercentErr: NaN,
+      httpPercentReq: NaN,
       isMTLS: '-1',
       protocol: undefined,
       responses: undefined,
-      responseTime: 'NaN',
-      tcp: 'NaN'
+      responseTime: NaN,
+      tcp: NaN
     },
     nodes: {
       app: undefined,
       destServices: undefined,
-      grpcIn: 'NaN',
-      grpcInErr: 'NaN',
-      grpcOut: 'NaN',
+      grpcIn: NaN,
+      grpcInErr: NaN,
+      grpcOut: NaN,
       hasCB: undefined,
       hasMissingSC: undefined,
       hasVS: undefined,
-      httpIn: 'NaN',
-      httpIn3xx: 'NaN',
-      httpIn4xx: 'NaN',
-      httpIn5xx: 'NaN',
-      httpOut: 'NaN',
+      httpIn: NaN,
+      httpIn3xx: NaN,
+      httpIn4xx: NaN,
+      httpIn5xx: NaN,
+      httpOut: NaN,
       isDead: undefined,
       isGroup: undefined,
       isInaccessible: undefined,
-      isMisconfigured: undefined,
-      isOutside: undefined,
+      isMiscofigured: undefined,
+      isOutsde: undefined,
```

Comment:
Hmmm are these typos in the received JSON?

Reply:
Good catch, I didn't notice it.